commandapi-nms/commandapi-1.19-common/src/main/java/dev/jorel/commandapi/nms/NMS_1_19_Common.java

```diff
@@ -594,6 +658,7 @@ public CommandSender getSenderForCommand(CommandContext<CommandSourceStack> cmdC
 
 		Entity proxyEntity = css.getEntity();
 		CommandSender proxy = proxyEntity == null ? null : proxyEntity.getBukkitEntity();
+
```

Comment:
Unnecessary new line, please revert. I have a series of diffing tools that I use to compare NMS versions and this will trip it up.